Resources/Locale/en-US/thief/backpack.ftl

```diff
@@ -15,9 +15,9 @@ thief-backpack-button-deselect = Select [X]
 
 thief-backpack-category-chameleon-name = chameleon's kit
 thief-backpack-category-chameleon-description =
-    Includes a full set of clothing that contain
-    chameleon technology, allowing you to disguise
-    as pretty much anything on the station.
+    Includes a full set of clothing that contains chameleon technology,
+    Contains a chameleon projector to help disguise yourself as objects,
```

Comment:
Do you really have to directly mention the projector in the description?

Reply:
I don't think that's a problem

Reply:
> how else are people meant to know about it

Then list every item in the kit, how else will they know